backend/api_v2/api_deployment_views.py

```diff
@@ -88,6 +88,9 @@ def get(
             response_status = status.HTTP_200_OK
             if not include_metadata:
                 response.remove_result_metadata_keys()
+        if response.result_acknowledged:
+            response_status = status.HTTP_406_NOT_ACCEPTABLE
```

Comment:
@muhammad-ali-e why is 406 chosen here? Can we avoid 4xx errors since its not necessarily a client error since the request did succeed as expected?
Do you think 200 or 204 might be better? I noticed [208 Already reported](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/208) as well but apparently it needs to be used in the context of web DAV

Reply:
I think a 4xx is appropriate here. Since the client shouldn't send the request again for same transaction once the result is retrieved.